### PR TITLE
Feature/UTC-217-Newsroom grid

### PR DESCRIPTION
**Pull request by brittrich**
New potential style for the newsroom grid. It's a masonry card display. I played with other side by side layouts...just landed on this one as the most visually appealing (personal preference, of corse). A few things;

- I modified two tailwind config files...did I make these changes in the right spots?
- Is there a way to remove the "Read more..." from the end of the card description? I looked a bit at the view and may have just missed the option.

Screen Shot 2021-05-09 at 10 59 01 PM

I've got a start on trying to address the card image cropping issue on the highlights, but you are right...the image sizing is tricky due to the variability in incoming images. Is there any way to ask them to use standard image sizes on articles so we have something consistent to work with. At the very least only use horizontal images? If not, have we consider some sort of smaller floating image and text layout that would allow for more variability image height while keeping a standard width?

### CODE REVIEW

File: source/default/_patterns/00-protons/tailwind.tokens.css [hunk 1]
```diff
@@ -16,6 +16,18 @@ Sans-Serif - Google Fonts - Open Sans
 Monospace - Google Fonts - Source Code Pro
 */
 
+@layer utilities {
+  @variants responsive {
```
Comment (bmartinez287) [inline source/default/_patterns/00-protons/tailwind.tokens.css:20]: this is fancy but it looks great. Had to read the docs  ;)

**Comment by bmartinez287** [comment]
> I modified two tailwind config files...did I make these changes in the right spots?

it looks like its fine there.

> Is there a way to remove the "Read more..." from the end of the card description? I looked a bit at the view and may have just missed the option.

Yes, there's a few ways that can be done. That might have to be done on the migration script when i bring the data in. i have to figure out how to leave the button tag that surrounds that and its content out of the description field.  @brittrich

**Comment by bmartinez287** [comment]
I was able to reduce the extra markup including images but the read more is tricky because it sees it as regular text.

**Comment by brittrich** [comment]
> > I modified two tailwind config files...did I make these changes in the right spots?
> 
> it looks like its fine there.
> 
> > Is there a way to remove the "Read more..." from the end of the card description? I looked a bit at the view and may have just missed the option.
> 
> Yes, there's a few ways that can be done. That might have to be done on the migration script when i bring the data in. i have to figure out how to leave the button tag that surrounds that and its content out of the description field. @brittrich

Thanks for checking on these!

**Pull request merged**